### ISSUE TITLE
Lack of proper manual

### ISSUE THREAD

**Issue by IgorTerechtchenko** [No-Bling/DOTA#58]
It would be nice to have what each option does explained in README.md

**Comment by Mackzwellz** [comment]
Look no further: https://github.com/No-Bling/DOTA/issues/47#issuecomment-655584220

Some options are not fully complete yet. Right now, developer is working on improving them and other things. Once all of it is done, I think he'll update the readme.

**Comment by IgorTerechtchenko** [comment]
Ok, thanks
